mods/project-longhorn-caption-buttons.wh.cpp

```diff
@@ -0,0 +1,701 @@
+// ==WindhawkMod==
+// @id				project-longhorn-caption-buttons
+// @name 			Restore Windows 7/Vista Caption Buttons for Project: Longhorn
+// @description		This mod is used by Project: Longhorn to have windows vista styled caption buttons.
+// @version			0.1
+// @author			PepperMarioYT
+// @github			https://github.com/PepperMarioYT
+// @include			dwm.exe
+// @architecture	x86-64
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+# The leaker known as ImSwordQueen. The leaker of mods while sleeping
+## I, PepperMarioYT did not create this mod, instead it was created by Dulappy (https://github.com/Dulappy), i just modified it for Project Longhorn

```

Comment:
Does it have to be specific to Project Longhorn? What was modified? I believe that it will be more useful to have it not tied to a specific project, and just make the things that you modified configurable via settings. What do you think?

Also, a more detailed readme will help, with information about what it does, supported Windows versions and screenshots.

Reply:
> Does it have to be specific to Project Longhorn? What was modified? I believe that it will be more useful to have it not tied to a specific project, and just make the things that you modified configurable via settings. What do you think?
> 
> Also, a more detailed readme will help, with information about what it does, supported Windows versions and screenshots.

Ok, ill modify it in my fork